### ISSUE TITLE
Setup doesn't work

### ISSUE THREAD

**Issue by vh** [vh/docker-modx#4]
But it is possible to finish setup manually after containers are runned!

docker-compose.yml

```
web:
  image: modx
  links:
    - db:mysql
  ports:
    - 80:80
db:
  image: mariadb
  environment:
    MYSQL_ROOT_PASSWORD: example
  ports:
    - 3306:3306
```

Log:

```
(ERROR in xPDOConnection::connect @ /var/www/html/core/xpdo/xpdo.class.php : 3058) SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known

(ERROR in xPDOConnection::connect @ /var/www/html/core/xpdo/xpdo.class.php : 3058) SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known

(ERROR @ setup/index.php) Could not connect to database server: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known
```

**Comment by vh** [comment]
Check this issue - https://github.com/modxcms/revolution/issues/12274

**Comment by vh** [comment]
When trying to add `opcache` to Dockerfile it cannot be done even manually.
Error: Checking database: MODX could not create your database. Please manually create your database and then try again.

**Comment by Mark-H** [comment]
I think this would need to be resolved before moving on to #2 and #3... Seems kinda critical to have a functional install ;)

**Comment by vh** [comment]
Definitely. But at this moment I have no idea why this issue appears :)

**Comment by vh** [comment]
It seems this issue resolved for new MODX 2.5 and php 7.0
